src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMSource.java

```diff
@@ -976,6 +1028,8 @@ protected boolean isCategoryEnabled(@NonNull SCMHeadCategory category) {
         StandardCredentials credentials = Connector.lookupScanCredentials(getOwner(), apiUri, scanCredentialsId);
         GitHub hub = Connector.connect(apiUri, credentials);
         GHRepository repo = hub.getRepository(getRepoOwner() + '/' + repository);
+        repositoryUrl = repo.getHtmlUrl();
```

Comment:
However the next line was a bug as the next line triggers a remote request (oh thank you GitHub API by KK for not clearly delineating the methods that cause a remote request from the methods that do not